source/blog/2018-07-16-ember-3-3-released.md

```diff
@@ -0,0 +1,113 @@
+We implemented a deprecation that removes access from `jQuery.Event#originalEvent`
+Read the full RFC: [emberjs/rfcs#294](https://github.com/emberjs/rfcs/blob/master/text/0294-optional-jquery.md).
+
+**deprecation of three private, unused classes (2 of 3)**
+Implemented deprecation of `Ember.Map`, `Ember.MapWithDefault`, and `Ember.OrderedSet` because not only were they private classes, they had also remained unused in Ember for some time now. Read the full (rendered) RFC: [emberjs/rfc#237](https://github.com/emberjs/rfcs/blob/master/text/0237-deprecation-ember-map.md)
+
+**deprecation of copy/copyable (3 of 3)**
+We've implemented deprecation of `Ember.copy` and `Ember.Copyable` - the `copy` function and `Copyable` mixin of `@ember/object/internals`. Ember isn't using it internally- it's leftover from SproutCore(!), and we're at the point where we realized we could clean this one up. 
+
+Shallow copies of the form `copy(x)` or `copy(x, false)` can be replaced mechanically with `Object.assign({}, x)`. 
+
+If you're using it in your addon, you might want to read the entire text of the RFC for advice on how to replace this in your addon: [emberjs/rfcs#322](https://github.com/emberjs/rfcs/blob/master/text/0322-deprecate-copy-copyable.md).
+
+For more details on changes in Ember.js 3.3, please review the [Ember.js 3.3.0 release page](https://github.com/emberjs/ember.js/releases/tag/v3.3.0).
+
+---
+
+## Ember Data
+
+Ember Data is the official data persistence library for Ember.js applications.
+
+### Changes in Ember Data 3.3
+
+Since Ember Data 3.2 was just released two weeks, ago we did not feel confident that the changes in the beta branch had received enough
+testing. As a result, we have decided to re-release Ember Data 3.2 as Ember Data 3.3 to maintain the release train cadence. Ember Data
+3.3.0-beta.1 will be released again as 3.4.0-beta.1 so it can continue to be evaluated for a full beta cycle.
+
+#### New Features (0)
+
+There are no new features in Ember Data 3.3.0.
+
+#### Deprecations (0)
+
+There are no deprecations introduced in Ember Data 3.3.
+
+For more details on changes in Ember Data 3.3, please review the
+[Ember Data 3.3.0 release page](https://github.com/emberjs/data/releases/tag/v3.3.0).
+
+---
+
+## Ember CLI
+
+Ember CLI is the command line interface for managing and packaging Ember.js applications.
+
+### Upgrading Ember CLI
+
+You may upgrade Ember CLI separately from Ember.js and Ember Data! To upgrade your projects using `yarn` run:
+
+```bash
+yarn upgrade ember-cli
+```
+
+To upgrade your projects using `npm` run:
+
+```bash
+npm install --save-dev ember-cli
+```
+
+After running the upgrade command run `ember init` inside of the project directory to apply the blueprint changes. You can preview those changes for [applications](https://github.com/ember-cli/ember-new-output/compare/v3.2.0...v3.3.0) and [addons](https://github.com/ember-cli/ember-addon-output/compare/v3.2.0...v3.3.0).
```

Comment:
Now that [`ember-cli-update`](https://github.com/ember-cli/ember-cli-update/) has been battle-tested, I think we should recommend using it for upgrading Ember applications (I recently updated [ember-cli release notes](https://github.com/ember-cli/ember-cli/releases/tag/v3.3.0) to reflect that.)

I think something along the lines:

> After running the upgrade command, make sure to install (if you haven't already) [`ember-cli-update`](https://github.com/ember-cli/ember-cli-update/) globally:

```bash
npm install -g ember-cli-update
```

> This utility will help you to update your app or add-on to the latest Ember CLI version. You will probably encounter merge conflicts, in which the default behavior is to let you resolve conflicts on your own. You can supply the `--resolve-conflicts` option to run your system's git merge tool if any conflicts are found.

thoughts?

Reply:
That's true, `--resolve-conflicts` is usually a hassle. My thinking was that since `ember-cli-update` is git-based people might want to use their own merge tool (save them some time and not look for documentation.) We can omit 😃